radixdlt/src/main/java/com/radixdlt/consensus/tempo/TempoException.java

```diff
@@ -30,8 +30,4 @@ public TempoException(String message) {
 	public TempoException(String message, Throwable cause) {
 		super(message, cause);
 	}
-
-	public TempoException(Throwable cause) {

```

Comment:
Constructor never used, and probably should require all exceptions to have some kind of context message anyway.